### ISSUE TITLE
LinearRegression Example in ForwardAccumulator docstring has an error

### ISSUE THREAD

**Issue by nickhalmagyi** [tensorflow/tensorflow#46848]
## URL(s) with the issue:

https://github.com/tensorflow/tensorflow/blob/85c8b2a817f95a3e979ecd1ed95bff1dc1335cff/tensorflow/python/eager/forwardprop.py#L234

## Description of issue (what needs changing):

I was trying to understand precisely how ForwardAccumulator.jvp works and was working through the example to which I have linked. From computing by hand what I believed to be the correct jvp formula, and comparing it to the linear regression code I found that in the line

  ...   loss = tf.reduce_sum((dense(x) - tf.constant([1., -1.])) ** 2.)

the target 
tf.constant([1., -1.]) 
which I would call 'y', should be reshaped, for example replaced by 
tf.constant([[1.], [-1.]]) 

The code as it stands introduces a multiplicative factor of n in the loss function where n is the number of samples. This same problem occurs in other places in this docstring. 

Would it be possible to include somewhere in this docstring a formula for the jacobian-vector-product?

**Comment by allenlavoie** [comment]
Nice catch, I have a fix for the broadcasting issue pending.

> Would it be possible to include somewhere in this docstring a formula for the jacobian-vector-product?

Are you referring to a formula for the JVP of one of the examples, or just JVPs in general?

**Comment by nickhalmagyi** [comment]
> Are you referring to a formula for the JVP of one of the examples, or just JVPs in general?

well I thought just in general, write the formula for the JVP. Its a very simple formula but oddly enough I found it hard to find a precise reference. Im not sure how latex is rendered in the documentation environment however. I know github is a bit slack with latex.
